README.md

```diff
@@ -27,6 +28,8 @@ This workflow is the current production workflow, equivalent to this [Cavatica p
 [STAR-Fusion v1.5.0](https://doi:10.1101/120295) Fusion detection for `STAR` chimeric reads.
 ### Arriba
 [Arriba v1.1.0](https://github.com/suhrig/arriba/) Fusion caller that uses `STAR` aligned reads and chimeric reads output.
+### AnnoFuse
```

Comment:
`annoFuse`
It's all about the brand bruh.

Reply:
Apple would never.